static/canjs.js

```diff
@@ -25,6 +25,10 @@ var $articleContainer,
 	hasShownSearch;
 
 (function() {
+
+	// Google Analytics
+	ga('send', 'pageview', window.location.pathname);
```

Comment:
@chasenlehara don't we already send the initial page load here: https://github.com/canjs/bit-docs-html-canjs/blob/4610356cb1c509fa784cb3ae7b5ba871e92664ab/templates/layout.mustache#L29